config/local_restrictions.yml

```
@@ -1557,165 +1563,201 @@ E07000094:
     - alert_level: 3
       start_date: 2020-12-19
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000242:
   name: East Hertfordshire District Council
   restrictions:
-    - alert_level: 2
-      start_date: 2020-12-02
-      start_time: "00:01"
     - alert_level: 3
       start_date: 2020-12-19
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000243:
   name: Stevenage Borough Council
   restrictions:
-    - alert_level: 2
-      start_date: 2020-12-02
-      start_time: "00:01"
     - alert_level: 3
       start_date: 2020-12-19
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000105:
   name: Ashford Borough Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000106:
   name: Canterbury City Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000107:
   name: Dartford Borough Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000108:
   name: Dover District Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000109:
   name: Gravesham Borough Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000110:
   name: Maidstone Borough Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000111:
   name: Sevenoaks District Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000112:
   name: Folkestone and Hythe
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000113:
   name: Swale Borough Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 4
+      start_date: 2020-12-21
+      start_time: "00:01"
 E07000114:
   name: Thanet District Council
   restrictions:
     - alert_level: 3
       start_date: 2020-12-02
       start_time: "00:01"
+      - alert_level: 4
+        start_date: 2020-12-21
+        start_time: "00:01"
```

This is indented one level too far.

```suggestion
    - alert_level: 4
      start_date: 2020-12-21
      start_time: "00:01"
```

---

good spot, thanks